lib/linter-eslint.coffee

```diff
@@ -44,7 +53,7 @@ module.exports =
 
   provideLinter: ->
     provider =
-      grammarScopes: ['source.js', 'source.js.jsx', 'source.babel', 'source.js-semantic']
+      grammarScopes: atom.config.get('linter-eslint.fileScopes').split(',')
```

Comment:
Even if he gets everytime, the main function is invoked only once. User will have to deactivate/activate package for it to take effect. He needs to store `grammarScopes` pointer somewhere and then pass it to linter